python/hyperspace/hyperspace.py

```diff
@@ -0,0 +1,174 @@
+import sys
+
+from pyspark.sql import SparkSession, DataFrame
+from py4j.java_gateway import java_import
+
+from .indexconfig import *
+
+
+class Hyperspace:
+    def __init__(self, spark):
+        """
+        Initializes Hyperspace object.
+        :param spark: sparkSession object
+        :return: Hyperspace object
+
+        >>> hyperspace = Hyperspace(spark)
+        """
+        self.spark = spark
+        self.jvm = spark._jvm
+        self.hyperspace = self.jvm.com.microsoft.hyperspace.Hyperspace(spark._jsparkSession)
+
+    def _getJavaIndexConfig(self, index_config):
+        """
+        Constructs IndexConfig Java object from python wrapper IndexConfig object.
+        :param index_config: IndexConfig java object
+        :return: IndexConfig python object
+
+        >>> _getJavaIndexConfig(idx_config)
+        """
+        indexed_columns = self._getScalaSeqFromList(index_config.indexedColumns)
+        print(indexed_columns)
```

Comment:
wait, are we printing this when you call this function?

Reply:
sorry these print statements was recently added for debugging efforts in build run. Forgot to remove this occurrence.